pkg/webhook/server/tls/dynamic_source.go

```diff
@@ -264,6 +268,11 @@ func (f *DynamicSource) updateCertificate(pk crypto.Signer, cert *x509.Certifica
 	// renew the certificate 1/3 of the time before its expiry
 	nextRenew <- cert.NotAfter.Add(certDuration / -3)
 
-	f.log.V(logf.InfoLevel).Info("Updated serving TLS certificate")
+	if f.cachedCertificate != nil && f.cachedCertificate.Leaf != nil {
+		f.log.V(logf.InfoLevel).Info("Updated cert-manager webhook TLS certificate", "DNSNames", f.DNSNames, "Subject", f.cachedCertificate.Leaf.Subject)
+	} else {
+		f.log.V(logf.InfoLevel).Info("Updated cert-manager webhook TLS certificate", "DNSNames", f.DNSNames)
+	}
```

Comment:
@irbekrm I found that for my local testing, `f.cachedCertificate.Leaf` was nil; hence I put this guard around the logging. I don't fully understand why the leaf is nil here.

Reply:
Another alternative would be to use: 
```
	leaf, err := x509.ParseCertificate(f.cachedCertificate.Certificate[0])
	if err == nil {
		f.cachedCertificate.Leaf = leaf
	}
```

In my tests, all those values are `null`, so I'm not sure it's worth parsing?

Reply:
If it's always `null`, then don't handle it.

Although hopefully someone can confirm it's always null.

Reply:
Running locally, I'm seeing stuff like this in the logs if I print out the Leaf's subject: 
```
I0523 18:25:26.727362       1 dynamic_source.go:276] cert-manager/webhook "msg"="Updated cert-manager webhook TLS certificate"  "DNSNames"=["cert-manager-webhook","cert-manager-webhook.cert-manager","cert-manager-webhook.cert-manager.svc"] "Subject"={"Country":null,"Organization":null,"OrganizationalUnit":null,"Locality":null,"Province":null,"StreetAddress":null,"PostalCode":null,"SerialNumber":"","CommonName":"","Names":null,"ExtraNames":null}
```

Reply:
Subject seems useless. Let's drop it. DNS names seems like it'd make it easier for someone to understand the source of the message/relevance.

Reply:
Ok @jsoref, I removed the subject since it didn't seem to have any meaningful information.